api/envoy/extensions/filters/http/set_metadata/v3/set_metadata.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.set_metadata.v3;
+
+import "google/protobuf/struct.proto";
+
+import "udpa/annotations/migrate.proto";
```

Comment:
Probably don't need `migrate.proto`

Reply:
Should be fixed in https://github.com/envoyproxy/envoy/pull/16400/commits/9d26ca29521869cfb630b3e7f02d271b7ff142bc